assets/js/app.js

```diff
@@ -1,18 +1,54 @@
 var $ = require('jquery');
 var _ = require('underscore');
+var socketio = require('socket.io-client');
+var tabs = require('./vendor/jquery.tabs');
+var sailsIOClient = require('sails.io.js');
 
-var view = require('./views/dashboard');
+var view = require('./views/centre');
 
 $(function() {
-  var options = {};
+  var io;
+  var socket0;
+  var container = $('#centres');
   var params = document.location.search.replace("?","").split("&");
-  var endpointParam = _.find(params, function(item) {
-    return item.match('endpoint');
+  var simulatorParam = _.find(params, function(item) {
+    return item.match('simulator');
   });
 
-  if (endpointParam) {
-    options.url = endpointParam.split("=")[1];
+  if (simulatorParam) {
+    var simulator = require('./socketSimulator');
+    socketio = simulator.client;
   }
 
-  new view(options);
+  io = sailsIOClient(socketio);
+
+  io.sails.autoConnect = false;
+  if (!simulatorParam) io.sails.url = 'http://localhost:8080';
+
+  socket0 = io.sails.connect();
+
+  socket0.get('/centre', function serverResponded(payload) {
+    _.each(payload, function(item,idx) {
+      container.append('<div id="item'+idx+'"></div>');
+
+      var thisView = new view({
+        el: '#item'+idx,
+        socket: socket0
+      });
+
+      thisView.model.set(item);
+    })
+
+  });
+
+  var $container = $('section.tabstuff');
+
+  if ($container.find('.js-tabs').length) {
```

Comment:
Thanks @chrisns for the comments - given the revised MVP, the tabs are effectively redundant, I'll therefore be removing the code. This is admittedly scrappy code, so I agree with you.